guides/micronaut-security-x509/metadata.json

```
@@ -0,0 +1,26 @@
+{
+  "title": "x.509 Mutual Authentication",
+  "intro": "Learn how to implement Mutual Authentication using SSL and X.509 certificates.",
```

```suggestion
  "intro": "Learn how to implement mutual authentication using SSL and X.509 certificates.",
```
Mutual authentication isn't a proper noun, so it doesn't need to be capitalized.